lib/sidekiq_jobs.rb

```diff
@@ -52,6 +53,24 @@ def perform(filename)
       end
     end
 
+    class Concordance
+      include Sidekiq::Job
+      def perform(filename_or_date)
+        # TODO: should we wrap this whole thing in a Sequel transaction?
```

Comment:
I think the answer is yes?

Reply:
Actually no, we don't want the extra headache. (The tests start failing.) If something goes wrong we fix and re-run.